test/e2e/framework/helpers/yunikorn/yk_utils.go

```diff
@@ -19,34 +19,26 @@
 package yunikorn
 
 import (
-	"errors"
 	"fmt"
-	"regexp"
 
 	"github.com/apache/incubator-yunikorn-k8shim/test/e2e/framework/configmanager"
 )
 
 type ResourceUsage struct {
-	memory string
-	vCPU   string
+	memory int64
+	vCPU   int64
```

Comment:
Really just a nit... do these have to be `int64`? Can't we just use normal `int`?

Reply:
Yes they really do. Int only allows for 4 GB RAM for example. All resources internally are mow 64-bit quantities.